### DISCUSSION TITLE
Self-Resolve Pull Request Conversations

### DISCUSSION THREAD

**Discussion by TheGiraffe3** [github/roadmap#1072] [Announcements]
My apologies if I'm not supposed to be submitting something here, but the Community Discussions are somewhat flooded. I asked a few months ago but haven't received a response from the team. https://github.com/orgs/community/discussions/128519
Below is the text of that discussion.

### Topic Area

Feature Request

### Body

It would be really nice if you could resolve a review comment you made. Otherwise, it’s a constant burden on the person making the PR to keep having to resolve, resolve, resolve.

**Comment by ankneis** [comment]
Hi @TheGiraffe3! Thanks for your participation - we will be closing this Discussion post. The [Community discussions](https://github.com/orgs/community/discussions) are the ideal place to raise feature requests, share feedback, and connect with fellow members. With a dedicated team of community managers and active product managers closely monitoring the forums this platform ensures better engagement and quicker responses compared to the roadmap discussions.
